platforms/platform_db/discrete_pcie3.json

```diff
@@ -30,6 +30,7 @@
             "params":
                {
                   "vc-supported": "{ 1, 0, 1, 0 }",
+                  "num-timing-reg-stages": 2,
                   "max-bw-active-lines-c0": "{ 256, 256, 256, 256 }",
                   "max-bw-active-lines-c1": "{ 128, 128, 128, 128 }"
```

Comment:
what are these values? perhaps add comments?

Reply:
These are documented in platform_defaults.json in the same directory.

Reply:
Ok